dist/api_data.py

```diff
@@ -55,20 +55,30 @@ def __ge__(self, other):
             this option is no longer supported, retained for backward compatibility''',
             choices=['always', 'never', 'none'], undoc=True),
         Config('read_timestamp', 'none', r'''
-            verify that timestamps should \c always or \c never be used on
-            reads with this table.  Verification should be set to \c none
-            if mixed read use is allowed''',
+            check that timestamps are \c always or \c never used on reads
+            with this table, failing on error;  should be set to \c none if
+            mixed read use is allowed''',
             choices=['always', 'never', 'none']),
         Config('write_timestamp', 'off', r'''
-            verify that commit timestamps are used per the configured
-            \c write_timestamp_usage option for this table''',
+            check that timestamps are used per the configured
+            \c write_timestamp_usage option for this table, failing on
+            error''',
+            choices=['off', 'on']),
+        ]),
+    Config('verbose', '', r'''
+        enable enhanced timestamp checking with messages''',
+        type='category', subconfig= [
+        Config('read_timestamp', 'none', r'''
```

Comment:
I like the change of bringing the read_timestamp also into the verbose. Just a thought is it good if we have a `read_timestamp_usage` similar to `write_timestamp_usage`?

Reply:
I thought about that too, but decided it didn't make a difference since `always` or `none` were the only configurations I could think of that would make sense for `read_timestamp_usage`, so it wouldn't make anything simpler to add that configuration string.

If you think there are more configurations that are interesting, I'm happy to talk through that change.